dev/com.ibm.ws.install.featureUtility_fat/fat/src/com/ibm/ws/install/featureUtility/fat/InstallServerTest.java

```diff
@@ -137,20 +137,51 @@ public void testInstallBlankFeatures() throws Exception {
 
 
     /**
-     * Install jsf-2.2 on its own first, then install a server.xml with jsf-2.2 and cdi-1.2. The autofeature cdi1.2-jsf2.2 should be installed along with the other features.
+     * Install osgiConsole-1.0 on its own first, then install a server.xml with osgiConsole-1.0 and eventLogging-1.0. The autofeature osgiConsole-1.0-eventLogging-1.0 should be installed along with the other features.
      * @throws Exception
      */
     @Test
     public void testInstallAutoFeatureServerXml() throws Exception {
```

Comment:
This test was consistently failing due to connection problem. In InstallFeatureTest, made changes to instead test for osgiConsole and eventLogging earlier. For server test class, made similar changes as dependency is low for these features and to remain consistent throughout the class.